### ISSUE TITLE
⌨ Overlap of collapse focus background with action links

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#11975]
## ⌨ Accessibility Issue

The links underneath a collapse element overlap with the triggers focus background and looks odd making things difficult to read.

## 🦋 Expected Behaviour

No overlap of background colour and action links for a collapsible item.

## 📋 Steps to Reproduce

1. Login as manager `manager@test.com`
2. Navigate to the manager dashboard
3. Focus the "Your talent requests" trigger
4. Observe the focus background overlap

## 📸 Screenshot

![Screenshot 2024-11-14 155715](https://github.com/user-attachments/assets/36300380-a67c-4f9c-8588-34b905c48a74)

## 🙋‍♀️ Proposed Solution

1. Add more space between the two items
2. Or, find a way to reduce the background colour coverage

## ✅ Acceptance Criteria

- [ ] No overlap between focus background and action links
- [ ] Little to no visual changes outside of fixing overlap
- [ ] While we're here change focus colour of text to black
- [ ] Design review

**Comment by tristan-orourke** [comment]
Spacing should be the amount of space here between the trigger and the buttons under the subtitle. This is just a case with no subtitle.

![Image](https://github.com/user-attachments/assets/5a4e8f13-a120-482a-9205-98866c4dbc74)

From https://www.figma.com/design/oZhQe9ahidHFfaiqgREvxa/Dashboard-(All-users)?node-id=2076-106797&node-type=frame&t=T1UxtPSdbTDiUQBL-0
